spec/system/community/community_spec.rb

```diff
@@ -468,27 +480,75 @@
 
     expect(page).to_not have_text(topic_2.title)
     expect(page).to have_text(topic_1.title)
-  end
 
-  scenario 'user searches for topics in community by post body content' do
-    # Let's set the titles for both topics to completely different sentences to avoid confusing the fuzzy search algo.
-    topic_1.update!(title: 'Complex sentence with certain words')
-    create :post,
-           topic: topic_3,
-           creator: student_1.user,
-           post_number: 2,
-           body: 'Another Complex Sentence'
+    find("button[title='Remove selection: complex sentence']").click
 
-    sign_in_user(coach.user, referrer: community_path(community))
+    fill_in 'filter', with: 'Completely Different'
 
+    click_button 'Search by content: Completely Different'
+
+    expect(page).to_not have_text(topic_1.title)
     expect(page).to have_text(topic_2.title)
 
+    find("button[title='Remove selection: Completely Different']").click
+
+    # Apply solved filter
+    fill_in 'filter', with: 'complex sentence'
+
+    click_button 'Search by title: complex sentence'
+
+    fill_in 'filter', with: 'solution'
+
+    click_button 'Solution: Solved'
```

Comment:
Why all these blank lines between test _user actions_? While I sometimes leave blank lines between _user actions_ and _test expectations_, this spacing seems a bit excessive. 🤔